src/configs/arbitrum/config.json

```diff
@@ -20,57 +20,6 @@
-      "name": "Contributor Proposal",
-      "title": "[NAME] Contributor Proposal [DATE - DATE]"
-    },
-    {
-      "description": "# Timeframe\n \n - State the contributor proposal date range\n \n # Background\n \n - State the context for making the contributor payment proposal, including:\n - What position you will be filling – i.e. developer, marketing, governance, etc.\n - Proposed Scope of Contribution\n - Self-Reflection on Scope of Contribution\n - Include the URL to the original proposal forum post\n \n # Deliverables\n\n - Include information around deliverables that were completed during your worker period\n \n # Compensation\n\n - Experience Level : 1-5\n - Time commitment : Percent (%) of time\n - [ETH/DAI AMT] : For second half payment proposal\n - [DXD AMT] : Vested for 2 years with a 1 year cliff starting the [DATE WORK PERIOD STARTS]\n - [% AMT REP]: For 2 second half payment proposal\n \n # Work Experience\n \n - Prior DXdao work proposals (if applicable)\n - Past Work for DXdao\n - If a developer, Github links\n - If a non-tech worker, doc links or described\n - If new, past relevant work experience\n - Developers : Github profile link\n - Non-tech : suggested Linkedin / other references\n \n",
-      "name": "Payment Proposal",
-      "title": "[NAME] Payment Proposal [DATE - DATE]"
-    },
-    {
-      "description": "## Description \n\n## Proposition \n\n## Action Plan",
-      "name": "Signal Proposal",
-      "title": "Signal Proposal for [DAO_ACTION]"
-    },
-    {
-      "calls": [
-        {
-          "functionName": "approve(address,uint256)",
-          "params": [
-            "0xecA7F78d59D16812948849663b26FE10E320f80C",
-            ""
-          ],
-          "to": "0xdE903E2712288A1dA82942DDdF2c20529565aC30"
-        },
-        {
-          "functionName": "createDistribution(address[],address,uint256[],uint64,uint64,bool,uint256)",
-          "params": [
-            "[0xdE903E2712288A1dA82942DDdF2c20529565aC30]",
-            "",
-            "[]",
-            "",
-            "",
-            "false",
-            "0"
-          ],
-          "to": "0xecA7F78d59D16812948849663b26FE10E320f80C"
-        }
-      ],
-      "description": "Simple description",
-      "name": "SWPR Farming Campaign",
-      "title": "[PAIR X/Y] Farming Campaign [DATE - DATE]]"
```

Comment:
Dont we need this still?

Reply:
I removed cause it was not use in the dapp, I couldnt find any place where the proposalTemplates array was used/

Reply:
Oh, think I was getting mixed up with another bit of config